tests/features/project.feature

```diff
@@ -16,6 +16,7 @@ Feature: Project content creation
     And the following document:
       | name          | file       |
       | My Document 1 | sample.pdf |
+      | My Document 2 | sample.pdf |
```

Comment:
We should probably have separate file names for this so we can differentiate them after.

Reply:
We have dedicated regions https://github.com/openeuropa/oe_content/blob/74b5f8dd7cde1f56fc32963525e565294101492d/behat.yml.dist#L69 which ensure that the values are correctly assigned, would you agree that, taking above into consideration, its better to reuse existing files then adding more to the fixtures ?

Reply:
The problem is that because we check for "sample.pdf" in both cases, with this test we can never be sure about which of the two media items is being rendered. I don't think adding another fixture would make much of an impact? What to you think @ademarco ?

Reply:
It would be better to use a different file, even if we have a dedicated region, as we aren't technically sure we are rendering the document "My Document 2" (that's what @imanoleguskiza is trying to say, I think).

Simply add another sample file to our fixture directory, commit it and use that one.